From beba17f43925823308c6f7f0d5ca9b52d00d351f Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Tue, 3 Nov 2020 18:34:20 -0800
Subject: [PATCH] Fix mistaken Belize interpretation

* northamerica (Belize): DST was in effect during most of
World War II (thanks to P Chan).
---
 northamerica | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/northamerica b/northamerica
index 00b2e6d..6a66617 100644
--- a/northamerica
+++ b/northamerica
@@ -3002,9 +3002,9 @@ Zone America/Barbados	-3:58:29 -	LMT	1924 # Bridgetown
 
 # Rule	NAME	FROM	TO	-	IN	ON	AT	SAVE	LETTER/S
 Rule	Belize	1918	1941	-	Oct	Sat>=1	24:00	0:30	-0530
-Rule	Belize	1919	1945	-	Feb	Sat>=8	24:00	0	CST
-Rule	Belize	1942	1944	-	Oct	Sat>=1	24:00	1:00	CWT
-Rule	Belize	1945	only	-	Oct	Sat>=1	24:00	1:00	CDT
+Rule	Belize	1919	1942	-	Feb	Sat>=8	24:00	0	CST
+Rule	Belize	1942	only	-	Jun	27	24:00	1:00	CWT
+Rule	Belize	1945	only	-	Aug	14	23:00u	1:00	CPT
 Rule	Belize	1945	only	-	Dec	15	24:00	0	CST
 Rule	Belize	1947	1967	-	Oct	Sat>=1	24:00	0:30	-0530
 Rule	Belize	1948	1968	-	Feb	Sat>=8	24:00	0	CST
-- 
1.8.3.1

